ui/package.json

```diff
@@ -24,9 +24,11 @@
     "url": "https://github.com/apache/cloudstack/issues"
   },
   "scripts": {
+    "prebuild": "./prebuild.sh",
     "start": "vue-cli-service lint --no-fix && vue-cli-service serve",
     "serve": "vue-cli-service lint --no-fix && vue-cli-service serve",
     "build": "vue-cli-service build",
+    "postbuild": "./postbuild.sh",
```

Comment:
@shwstppr could relative paths cause any issue? For ex. break in deb/rpm builds?